src/pages/ServiceList/ServiceListComponent.tsx

```diff
@@ -63,9 +62,9 @@ export const sortFields: SortField[] = [
     title: 'Error Rate',
     isNumeric: true,
     compare: (a: ServiceItem, b: ServiceItem) => {
-      const ratioA = getRequestErrorsRatio(a.health.requests);
-      const ratioB = getRequestErrorsRatio(b.health.requests);
-      return ratioA === ratioB ? a.name.localeCompare(b.name) : ratioA.value - ratioB.value;
+      const ratioA = getRequestErrorsRatio(a.health ? a.health.requests : undefined).value;
```

Comment:
Isn't more clear to deal directly with 0/-1 here, instead to delegate into the getRequestErrorRation() to resolve the "undefined" value ?
I saw that logic hard to review in a first attemp.

Reply:
+1